.github/workflows/nightly.yml

```
@@ -40,15 +41,9 @@ jobs:
     - name: Python tests, refreshing the network snapshots
       env:
         WK_TOKEN: ${{ secrets.WK_TOKEN }}
-      run: ./test.sh --refresh-snapshots
+      run: ./test.sh --refresh-snapshots --splits 5 --group ${{ matrix.group }}
```

Shouldn't this be 3?
```suggestion
      run: ./test.sh --refresh-snapshots --splits 3 --group ${{ matrix.group }}
```

---

Yep! This is also a problem in the normal CI action, fixing it there as well :see_no_evil: